lib/futurism/helpers.rb

```diff
@@ -1,13 +1,17 @@
 module Futurism
   module Helpers
-    def futurize(records, extends:, &block)
+    def futurize(extends, records, **options, &block)
```

Comment:
As I mentioned on discord, I'd like to defer this topic for now... could you maybe extract that hunk into its own branch?

Reply:
This branch isn't intended to be merged! Not yet, maybe not ever. I'm just keep it synced to my local dev

Reply:
Serious question tho: if I need to put a class on the placeholder TR, how else would I do this?

Right now, it's like a feature-limited implementation of content_tag...